### PR TITLE
Topbar improvements

### PR DESCRIPTION
**Pull request by aditudorache**

### CODE REVIEW

File: packages/asc-ui/src/components/Modal/Modal.stories.tsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ storiesOf('Modal', module)
       onClose={action('close modal from escape button or click outside')}
       blurredNode={window.document.querySelector('#root') as HTMLInputElement}
     >
-      <TopBar>
+      <TopBar backgroundColor="lightgrey">
```
Comment (royderks) [inline packages/asc-ui/src/components/Modal/Modal.stories.tsx]: Use colors from theme

File: packages/asc-ui/src/components/TopBar/TopBar.stories.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import Typography from '../Typography'
 import IconButton from '../IconButton'
 
 const SimpleComponent: React.FC<{}> = () => (
-  <TopBar>
+  <TopBar backgroundColor="grey">
```
Comment (royderks) [inline packages/asc-ui/src/components/TopBar/TopBar.stories.tsx]: Same, use theme colors

File: packages/asc-ui/src/components/TopBar/StyledTopBar.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components'
+
+const defaultBackgroundColor = '#fff'
```
Comment (royderks) [inline packages/asc-ui/src/components/TopBar/StyledTopBar.ts]: Use colors from theme and preferably create an object called defaultProps { ... }

File: packages/asc-ui/src/components/TopBar/StyledTopBar.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+const StyledTopBar = styled.header<Props>`
+  display: flex;
+  position: relative;
+  align-items: center;
+  z-index: 0;
+  min-height: 54px;
+  padding: 0 15px;
+  background-color: ${props =>
```
Comment (royderks) [inline packages/asc-ui/src/components/TopBar/StyledTopBar.ts]: Destructure

File: packages/asc-ui/src/components/Modal/Modal.stories.tsx (hunk 1 above)
Comment (aditudorache) [reply packages/asc-ui/src/components/Modal/Modal.stories.tsx]: done

File: packages/asc-ui/src/components/TopBar/StyledTopBar.ts (hunk 3 above)
Comment (aditudorache) [reply packages/asc-ui/src/components/TopBar/StyledTopBar.ts]: done

File: packages/asc-ui/src/components/TopBar/StyledTopBar.ts (hunk 4 above)
Comment (aditudorache) [reply packages/asc-ui/src/components/TopBar/StyledTopBar.ts]: done

File: packages/asc-ui/src/components/TopBar/TopBar.stories.tsx (hunk 2 above)
Comment (aditudorache) [reply packages/asc-ui/src/components/TopBar/TopBar.stories.tsx]: done

**Pull request merged**